index.js

```diff
@@ -66,11 +66,11 @@ function bootApp() {
   })
 
   app.get('/health', (req, res) => {
-    if (cache.isAvailable()) {
-      res.status(200).end()
-    } else {
-      res.status(503).end()
-    }
+    cache.isAvailable().then((stats) => {
```

Comment:
Do the `stats` hold any data that would influence the outcome or is any response instead of an error considered good enough?

Reply:
Just checking connectivity with this as there is no command analogous to Redis' `PING` see https://docs.oracle.com/cd/E17952_01/mysql-5.6-en/ha-memcached-stats.html but I don't think any of that is relevant to a service-level check